keylime/tpm/tpm_main.py

```diff
@@ -890,6 +890,8 @@ def verify_ek(self, ekcert):
         """
         # openssl x509 -inform der -in certificate.cer -out certificate.pem
         try:
+            tpm_ek_ca.check_tpm_cert_store()
+
```

Comment:
It seems that other than this new call `check_tpm_cert_store()` is unused before this. Strange...

But `check_tpm_cert_store()` calls `sys.exit()` when it fails. Are we sure that's what we want to do? Would it be better to throw an exception?

Reply:
Yes, it would be better to throw an exception. I'll make the changes.

I was surprised too when I found that check implemented but not called. I thought it was dropped at some point by accident, but it seems it was never used.